front/public/cards/disco.json

```diff
@@ -64,6 +65,27 @@
                     "g": 2
                 }
             }
+        },
+        {
+            "label": "GDC Sample Data",
+            "urlparam": "?genome=hg38&dslabel=GDC&disco=1&sample=TCGA-AP-A059-01A",
+            "runargs": {
+                "noheader": true,
+                "nobox": true,
+                "genome": "hg38",
+                "disco": {
+                    "dslabel": "GDC",
+                    "genome": "hg38",
+                    "sample_id": "TCGA-AP-A059-01A"
```

Comment:
the case is a hypermutator and slow to load. replace with TCGA-85-7710-01A

Reply:
@xzhou82 This is complete and the application list is updated as well.